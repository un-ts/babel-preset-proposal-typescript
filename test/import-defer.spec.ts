import { createRequire } from 'node:module'

import { execute } from './helpers.js'
import * as _ from './import-defer.cjs'

const cjsRequire = createRequire(import.meta.url)

const pkg = cjsRequire('@babel/core/package.json') as {
  version: string
}

const proposal = 'import-defer'

test(proposal, () => {
  expect(() => execute(proposal + '.cts')).toThrowErrorMatchingSnapshot()
  expect(_.getVersion()).toBe(pkg.version)
})
