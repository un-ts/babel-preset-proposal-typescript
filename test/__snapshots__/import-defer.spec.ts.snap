// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import-defer 1`] = `"Unexpected token '*'"`;
